app/assets/stylesheets/pages/settings.scss

```diff
@@ -110,3 +110,35 @@ $sidebar-item-highlight: #00c081;
 .ldap-conn-message-valid {
   color: $brand-success;
 }
+
+.cert-table {
+  td.field {
+    font-weight: bold;
+    width: 180px;
+    vertical-align: top;
```

Comment:
Could you add `padding-bottom: 3px` here?

Reply:
Done